src/scss/pages/_index.scss

```diff
@@ -0,0 +1,160 @@
+.main-content {
+    padding-bottom: $padding-bottom--48;
+    width: 100%;
+    margin-right: auto;
+    margin-left: auto;
+    padding-right: 20px;
+    padding-left: 20px;
+    .component-margin{
+        padding-top: $padding-top--48;
+    }    
+    .title-h2{
+        align-items: baseline;
+        .link-title{
+            color: $color-secondary;
+        }
+    }
+
+
+    .top-courses {
+        overflow: auto;
+        overflow-y: hidden;
+        scroll-snap-type: x mandatory;
+        scroll-snap-stop: always;
+        -ms-touch-action: manipulation;
+        touch-action: manipulation;
+        overflow-x: hidden;
+        display: grid;
+        grid-auto-flow: column;
+        grid-gap: $spacing--16;
+        grid-auto-columns: calc((100% - (5 - 1) * 16px)/1);
+
+        @media screen and (min-width: 37.56em) {
+            grid-auto-columns: calc((100% - (2 - 1) * 16px)/2);
+        }
+
+        @media screen and (min-width: 43.81em) {
+            grid-auto-columns: calc((100% - (3 - 1) * 16px)/3);
+        }
+
+        @media screen and (min-width: 61.31em) {
+            grid-auto-columns: calc((100% - (4 - 1) * 16px)/4);
+        }
+
+        @media screen and (min-width: 75.06em) {
+            grid-auto-columns: calc((100% - (5 - 1) * 16px)/5);
+        }
+
+        .item-courses {
+            display: flex;
+            align-items: flex-start;
+            flex-direction: column;
+            max-width: $max-width-375;
+            min-width: $min-width--173;
+
+            .thumb {
+                background-color: #1e1e1c;
```

Comment:
Why we need to use color here?
If you want to use it, please move it to colors.scss